Source/Core/Core/NetPlayServer.h

```diff
@@ -126,10 +132,13 @@ class NetPlayServer : public TraversalClientClient
 
   u64 GetInitialNetPlayRTC() const;
 
+  template <typename... Data>
+  void SendResponse(const MessageID message_id, const Client* player = nullptr,
```

Comment:
Ah... this default argument is what's causing problems with older versions of clang. The code is apparently valid, but [there was a bug.](https://bugs.llvm.org/show_bug.cgi?id=23029)

Personally, I'd split it into two versions, one for sending to all, one for sending to a single player. But you could also just remove default. 

The default argument doesn't make a huge amount of sense here anyway, as it can only be used when sending a message without data, and it forces the arguments into a weird order: message_id, destination, rest_of_message.

Reply:
Ah ok, I'll make two then :(